etc/config.xml

```diff
@@ -14,8 +14,10 @@
                 <retry_failures_cron>*/5 * * * *</retry_failures_cron>
                 <retry_failures_count>2</retry_failures_count>
                 <enable_auto_invoice>0</enable_auto_invoice>
-                <enable_rest_api_log>0</enable_rest_api_log>
             </fast>
+            <rest_api_log>
+                <enable_rest_api_log>0</enable_rest_api_log>
```

Comment:
So by default it is disabled ?

Reply:
Correct. This is how we had it. This update just puts it within the correct group (`<rest_api_log>`).